source/applicability/transport_of_data.rst

```diff
@@ -18,15 +18,75 @@ formatted text.
 
 Messages can be sent asynchronously, i.e. while the site or supervision
 system is waiting for an answer to a previously sent message it can
-can continue to send messages. The exception in the RSMP / SXL version message
-(see section :ref:`rsmpsxl-version`).
+can continue to send messages. The exception is during the first part of
+communication establishment (see section :ref:`communication-establishment-between-sites-and-supervision-system`
+and :ref:`communication-establishment-between-sites`).
+
+RSMP connections can be established:
+
+* Between site and supervision system.
+  See :ref:`communication establishment between sites and supervision system <communication-establishment-between-sites-and-supervision-system>`.
+  The site needs to support multiple RSMP connections to different
+  supervisors. See :ref:`Multiple supervisors <multiple-supervisors>`.
+
+* Directly between sites.
+  See :ref:`communication establishment between sites <communication-establishment-between-sites>`.
+
+.. note::
+   Implementing support for communication between sites is not required unless
+   otherwise stated in the :term:`SXL`.
+
+.. _multiple-supervisors:
+
+Multiple supervisors
+^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+   Implementing support for multiple supervisors is not required unless
+   otherwise stated in the :term:`SXL`.
+
+Each site needs to support the following:
+
+* At least three (3) RSMP connections to different supervision systems
```

Comment:
maybe we should move this number to the sxl, so different types of equipment can have differenrt requirements. (and of course a road authorityt can always 'override' with their own requirements)

Reply:
Fixed. Moved to SXL